consensus/chronology/chronology.go

```diff
@@ -116,6 +138,33 @@ func (chr *chronology) RemoveAllSubrounds() {
 	chr.mutSubrounds.Unlock()
 }
 
+func (chr *chronology) resetChronologyStuckAlarm() {
+	duration := chr.rounder.TimeDuration() * numRoundsToWaitBeforeSignalingChronologyStuck
+	alarmID := "chronologyStuck"
+
+	chr.alarmScheduler.Cancel(alarmID)
+
+	cb := func(alarmID string) {
+		buffer := new(bytes.Buffer)
+		err := pprof.Lookup("goroutine").WriteTo(buffer, 1)
```

Comment:
we can also print the stack trace for each go routine, if needed by using
```GO 
    buff := make([]byte, 1<<27)
    num := runtime.Stack(buff, true)
    dump := string(buff[:num])
```
This one looks exactly like the one we got when the code panics